sunpy/map/mapsequence.py

```diff
@@ -85,7 +89,154 @@ def __len__(self):
+                            elem.style.display = (elem.getAttribute("index") ==
+                                                      form.getAttribute("cur_index") ? "" : "none"
+                                                 );
+                        }}
+                    );
+                '/>
+
+                <!-- Button to increment index (starts enabled if there is more than one Map) -->
+                <input type=button value='&rarr;' style='font-weight: bold'
+                    {"" if len(repr_list) > 1 else "disabled"} onClick='
+
+                    var form = this.parentElement;
+
+                    // Increment index if allowed
+                    var cur_index = Math.min(
+                                        parseInt(form.getAttribute("cur_index")) + 1,
+                                        form.getAttribute("max_index")
+                                    );
+                    form.setAttribute("cur_index", cur_index);
+
+                    // Always enable the decrement button (because we just incremented)
+                    form.children[0].disabled = false;
+
+                    // Enable the increment button if and only if this is not the last Map
+                    form.children[1].disabled = (cur_index == form.getAttribute("max_index"));
+
+                    // Update string (which is children[2] of the form)
+                    form.children[2].innerHTML = "Map at index " + cur_index;
+
+                    // Update visibilities to show only the current index
+                    // This avoids for...of syntax to retain support for ES5 browsers (e.g., IE11)
+                    var array = Array.prototype.slice.call(form.lastElementChild.children);
+                    array.forEach(function (elem)
+                        {{
+                            var form = elem.parentElement.parentElement;
+                            elem.style.display = (elem.getAttribute("index") ==
+                                                      form.getAttribute("cur_index") ? "" : "none"
+                                                 );
+                        }}
+                    );
+
+                '/>
+
+                <!-- This string is updated as the index is changed -->
+                <span>Map at index 0</span>
+
+                <!-- This element is at the end so that lastElementChild will point to it -->
+                <div>
+                    {repr_list_html}
+                </div>
+            </form>""")
+
+    def quicklook(self):
+        """
+        Display a quicklook summary of the MapSequence instance using the default web browser.
+
+        Click on the ``<`` and ``>`` buttons to step through the individual maps.
+
+        Notes
+        -----
+        The image colormap uses
+        `histogram equalization <https://en.wikipedia.org/wiki/Histogram_equalization>`__.
+
+        Interactive elements require Javascript to be supported by the web browser.
```

Comment:
I tweaked the sentence so it says "...require Javascript support to be enabled in the web browser."  I actually did encounter an issue when testing in IE because IE was blocking Javascript execution by default.